tools/run_tests/generated/sources_and_headers.json

```diff
@@ -4295,6 +4295,24 @@
     "third_party": false, 
     "type": "target"
   }, 
+  {
+    "deps": [
+      "gpr", 
```

Comment:
Please take a look at load_file_test and trim the dependencies. Also, does language need to be c++?